tests/migrations/test_autodetector.py

```diff
@@ -2181,6 +2181,150 @@ def test_rename_model_reverse_relation_dependencies(self):
             changes, "testapp", 0, 0, old_name="EntityA", new_name="RenamedEntityA"
         )
 
+    def test_move_model_with_no_db_table(self):
+        before = [self.author_name, self.book]
+        after = [
+            ModelState(
+                "otherapp",
+                "Author",
+                [
+                    ("id", models.AutoField(primary_key=True)),
+                    ("name", models.CharField(max_length=200)),
+                ],
+            ),
+            ModelState(
+                "otherapp",
+                "Book",
+                [
+                    ("id", models.AutoField(primary_key=True)),
+                    ("author", models.ForeignKey("otherapp.Author", models.CASCADE)),
+                    ("title", models.CharField(max_length=200)),
+                ],
+            ),
+        ]
+        changes = self.get_changes(
+            before, after, MigrationQuestioner({"ask_move_model": True})
+        )
+        self.assertNumberMigrations(changes, "otherapp", 2)
+
+        self.assertMigrationDependencies(changes, "otherapp", 0, [])
```

Comment:
I guess a sense that this is wrong. The migration that creates the unmanaged models in `otherapp` should depend on the model already existing in `testapp`